stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -353,6 +353,13 @@ module SwiftGlibc [system] {
       }
 % end
 
+% if CMAKE_SDK in ["LINUX"]:
+    module xattr {
+        header "${GLIBC_ARCH_INCLUDE_PATH}/sys/xattr.h"
```

Comment:
Ahh, yes, you're right- wrong variable. My mistake. Looking at it, this would probably be better to go under the posix module anyway